app/scripts/background.js

```diff
@@ -293,6 +292,8 @@ async function loadStateFromPersistence() {
   // write to disk
   if (localStore.isSupported) {
     localStore.set(versionedData);
```

Comment:
Nit: Could we use `persistStateToLocalStore` here instead? That is all we're doing here, persisting state.

Then we can remove this `isSupported` check, and simplify this function a bit. We'd need to set the metadata first before calling `persistDateToLocalStore` though.

Reply:
Done [here](https://github.com/MetaMask/metamask-extension/pull/15978/commits/6d82b9026a036ba1b960d4e9659e73f542896dc7)